AndroidBound/src/main/java/solutions/alterego/androidbound/binding/BindingAssociationEngine.java

```diff
@@ -125,12 +132,16 @@ public void accept(Exceptional<Object> value) throws Exception {
                             }
                         });
             } else {
-                mLogger.warning("Binding " + mBindingSpecification.getSource()
-                        + " needs Disposable, but changes were not available");
+                String msg = "Binding " + mBindingSpecification.getSource() + " needs Disposable, but changes were not available = " + mBindingSpecification.toString();
```

Comment:
also, please avoid `+` for string concatenation and use `concat`

Reply:
I always thought that `+` gets converted to `StringBuilder` by the compiler, which is quicker when having bigger strings: https://stackoverflow.com/a/46485284

Reply:
brain fart - sorry